ahk/window.py

```diff
@@ -648,7 +648,9 @@ def win_get(self, title='', text='', exclude_title='', exclude_text='', encoding
             return None
         return Window(engine=self, ahk_id=ahk_id, encoding=encoding)
 
-    def win_wait(self, *, title='', text='', exclude_title='', timeout=0.5, exclude_text='', encoding=None):
+    def win_wait(
```

Comment:
I think rather than adding `exact` as a keyword argument, this should accept an argument for the matching mode to use. That way, it reflects the AHK API more clearly/understandably.

For example:

```py
TitleMatchMode: Literal[1, 2, 3]
TitleMatchSpeed: Literal["Slow", "Fast"]
...
def win_wait(..., match_mode: Optional[TitleMatchMode]=None, match_speed: Optional[TitleMatchSpeed] = None ...):
    ...
```
Type hints are just there for clarity in explaining what I'm thinking and don't necessarily need to be added.

Then you can pass either 1 or 2 additional parameters to the AHK side of things and do:
```jinja
{% if match_mode %}
SetTitleMatchMode, {{ match_mode }}
{% endif %}
{% if match_speed %}
SetTitleMatchMode, {{ match_speed }}
{% endif %}
```
and for the daemon something like
```jinja
match_mode := command[7]
match_speed := command[8]
if (...) {
    ...  ; set one or both of match mode/speed
}
```
Hope that makes sense. Let me know what you think.

Reply:
Thank you for your advice and feedback. This is helpful.